_posts/2024-02-07-#28950.md

```diff
@@ -8,16 +8,56 @@ authors: [instagibbs]
 components: ["rpc"]
 host: ismaelsadeeq
 status: upcoming
-commit:
+commit: bf6fbea9794db5e8d0adce8ff6182574808cea54
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
+
+- Package transactions submitted to the [`submitpackage`](https://bitcoincore.org/en/doc/26.0.0/rpc/rawtransactions/submitpackage/) RPC successfully are accepted to the node's mempool, but broadcasted to peers individually because [Package relay](https://bitcoinops.org/en/topics/package-relay/) is currently a work in progress. Successful submission does not mean the transactions will propagate.
```

Comment:
```suggestion
- Package transactions submitted to the [`submitpackage`](https://bitcoincore.org/en/doc/26.0.0/rpc/rawtransactions/submitpackage/) RPC successfully are accepted to the node's mempool, but broadcasted to peers individually because [package relay](https://bitcoinops.org/en/topics/package-relay/) is currently a work in progress. Successful submission does not mean the transactions will propagate.
```